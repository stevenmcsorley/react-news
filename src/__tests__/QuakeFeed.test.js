import React from 'react';
import { shallow } from 'enzyme';
import Quake from '../components/news/QuakeFeed'

test("testing Quake feed",  async () =>{

    const QuakeComp = shallow(<Quake />, 'div');
    expect(QuakeComp).toMatchSnapshot();

})