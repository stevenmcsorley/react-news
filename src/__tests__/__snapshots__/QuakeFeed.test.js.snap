// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing Quake feed 1`] = `ShallowWrapper {}`;
